nglview/utils/widget_utils.py

```diff
@@ -90,3 +90,42 @@ def func(this, **kwargs):
             func = make_func(rep)
             fn = '_'.join((root_fn, rep[0]))
             setattr(self, fn, MethodType(func, other))
+
+
+def compare_two_images(image_path_1:str,
+                       image_path_2:str,
+                       threshold=0.97) -> bool:
+    """
+    Comparing two images. Two images will be convert to new size
+    (standard_width, standard_height) and then compute structural similarity
+    to compare them.
+    Input:
+    ----------------
+    image_path_1: string, path for the first image
```

Comment:
you can remove all the docs here since you already annotate them and there's nothing new in the docs.